commitizen/commands/commit.py

```diff
@@ -89,7 +89,7 @@ def manual_edit(self, message: str) -> str:
         subprocess.call(argv)
         with open(file_path) as temp_file:
             message = temp_file.read().strip()
-        file.unlink()
+        os.unlink(file.name)
```

Comment:
Seems good to me, but maybe it would be cleaner to just move the `subprocess.call` into the context manager and remove the `delete=False` parameter.
It would avoid the temporary file remaining present in case of an exception during the `subprocess.call()` plus, from the [latest documentation](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile), it seems to properly handle some permissions issues that might happen while doing a manual unlink on Windows.

Reply:
sounds good. let me do the change and get this merged

Reply:
@noirbizarre after some testing, it doesn't seem to work that way 🤔 
@alxgu 's implementation works fine though

Reply:
I'll approve it and let you decide whether we should merge it

Reply:
Good to me 👍🏼

Reply:
niiice. let's merge it then

Reply:
Thank you for reviewing and merging 😃